cleaner.go

```diff
@@ -419,24 +416,29 @@ func main() {
 	if err != nil {
 		log.Err(err).Msg("Load configuration")
 	}
-
 	err = CheckConfiguration(&config)
 	if err != nil {
 		log.Err(err).Msg("Check configuration")
 		return
 	}
-
-	if config.Logging.Debug {
-		log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+	// override default value read from configuration file
```

Comment:
where does this come from? is it related to glitchtip?

Reply:
This was used to set logging mode to debug but when i was implementing ability to send alerts to sentry i used the zerologInit function witch already implements this so i removed that code to avoid duplicity

Reply:
I meant the maxAge parameter

Reply:
```
	if cliFlags.MaxAge != "" {
		config.Cleaner.MaxAge = cliFlags.MaxAge
	}
```

Reply:
I dont know why it is shown in the pr as it was already implemented before my PR https://github.com/RedHatInsights/insights-results-aggregator-cleaner/blob/3fd2f520e75955957da4785de6d2bc2c820ee27b/cleaner.go#L436C2-L439C1

Reply:
I probably moved it in the code i will put it where it was before